Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -34,7 +34,8 @@ PowerPCState ppcState;
 
 static CPUCoreBase* s_cpu_core_base = nullptr;
 static bool s_cpu_core_base_is_injected = false;
-Interpreter* const s_interpreter = Interpreter::getInstance();
+Interpreter interpreter;
+Interpreter* const s_interpreter = &interpreter;
```

Comment:
Any reason for keeping this around, given the instance is literally right above it? This isn't used outside of this file.

Reply:
You're right. I didn't even think about grepping for other uses.